openmicroscopy/openmicroscopy
# custom login logo in settings.py

This provides 2 ways to add a custom logo to the web login screen.
To test:
- replace static/webclient/image/custom_login_logo.png with your own image (named the same). Refresh the webclient.
- Choose a url to an image you want to use, and add this via config, E.g:

`$ bin/omero config set omero.web.login_logo "http://www.drugdiscovery.dundee.ac.uk/sites/www.drugdiscovery.dundee.ac.uk/files/ddu-final.png"`

Restart web - logo should appear on login screen - see screenshot:

![screen shot 2014-01-11 at 12 35 04](https://f.cloud.github.com/assets/900055/1895077/30edc57a-7b0f-11e3-8061-98c39e60b8ea.png)

NB: Need to add this to web docs.
